index.html

```diff
@@ -1,16 +1,269 @@
+<td><ul><li>Developed according to automotive requirements</li><li>Implementation based on static memory and lock-free algorithms</li><li>Huge C++ library with safe STL implementations</li></ul></td>
+</tr>
+</tbody>
+</table>
+
+
+<table align="center">
+<thead>
+<tr>
+<th colspan="2"><h2>Commercial support</h2></th>
+</tr>
+</thead>
+<tbody>
+<tr>
+<td align="center" valign="top" width="33%">
+  <a href="https://ekxide.io">
+    <img src="./images/ekxide-white-on-black-rounded.png" alt="ekxide IO GmbH">
+  </a><br />
+  <a href="mailto:info@ekxide.io">info@ekxide.io</a>
+</td>
+<td>
+  <ul>
+    <li>commercial extensions and tooling</li>
+    <li>custom feature development</li>
+    <li>training and consulting</li>
+    <li>integration support</li>
+    <li>engineering services around the iceoryx ecosystem</li>
+  </ul>
+</td>
+</tr>
+</tbody>
+</table>
+
+            </article>
+          </div>
+        </div>
+
+      </main>
+
+        <footer class="md-footer">
+    <div class="md-footer-meta md-typeset">
+        <div class="md-footer-meta__inner md-grid">
+            <div class="md-copyright">
+                Made with
+                <a href="https://squidfunk.github.io/mkdocs-material/" target="_blank" rel="noopener">
+                    Material for MkDocs
+                </a>
+            </div>
+        </div>
+        <div class="md-footer-meta__inner md-grid">
+            <div class="md-copyright">
+                Copyright © 2021 Eclipse Foundation<br>
+                <a href="https://www.eclipse.org/legal/privacy.php" target="_blank">Privacy policy</a><br>
+                <a href="https://www.eclipse.org/legal/termsofuse.php" target="_blank">Terms of use</a><br>
+                <a href="https://www.eclipse.org/legal/copyright.php" target="_blank">Copyright</a><br>
+                <a href="https://www.eclipse.org/legal/" target="_blank">Legal</a><br>
+                <a href="https://www.eclipse.org/security/" target="_blank">Report a safety or security
+                    vulnerability</a>
+            </div>
+        </div>
+        <div class="md-footer-meta__inner md-grid">
+            <div class="md-copyright">
+                Sponsored by<br>
+                <a href="https://www.eclipse.org/" target="_blank" rel="noopener" title="Eclipse"><img
+                        src="./images/eclipse-logo.svg" width="100px" style="margin-top: 0.5rem"></img></a><br>
+                <a href="https://www.bosch.com/" target="_blank" rel="noopener" title="Robert Bosch GmbH"><img
+                        src="./images/bosch-logo.png" width="100px" style="margin-top: 0.5rem"></img></a><br>
+                <a href="https://www.apex.ai/" target="_blank" rel="noopener" title="Apex.AI Inc."><img
+                        src="./images/apex-logo.webp" width="100px" style="margin-top: 0.5rem"></img></a><br>
+                <a href="https://www.ekxide.io/" target="_blank" rel="noopener" title="ekxide IO GmbH"><img
```

Comment:
:rocket: Please add this icon in a follow-up PR also to the `iceoryx1` sources.

Reply:
Sure. Btw, do you have a higher resolution Apex.AI logo or one in the SVG format?

Reply:
I can add one as part of https://github.com/eclipse-iceoryx/iceoryx/issues/1295